### PR TITLE
[AKS] Add option WindowsAnnual to --os-sku for az aks nodepool add

### PR DESCRIPTION
**Pull request by junjiezhang1997**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### Related command
`aks nodepool add --resource-group={resource_group} --cluster-name={name} --name={windows_nodepool_name} --node-count=1 --os-type Windows --os-sku WindowsAnnual`

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [x] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
AKS

**Review by AbelHu — APPROVED** [review]
LGTM. Thanks for the PR.

**Review by FumingZhang — APPROVED** [review]
lgtm

Author queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=92097207&view=results), test passed!
- test_aks_nodepool_add_with_ossku_windowsannual

File: src/aks-preview/azcli_aks_live_test/configs/ext_matrix_default.json [hunk 1]
```diff
@@ -56,6 +56,9 @@
         ],
         "automatic, missing feature registration & toggle": [
             "test_aks_automatic_sku"
+        ],
+        "windows annual, missing feature registration & toggle": [
+            "test_aks_nodepool_add_with_ossku_windowsannual"
```
Comment (FumingZhang) [inline src/aks-preview/azcli_aks_live_test/configs/ext_matrix_default.json:61]: don't forget to remove the exclusion after the feature is enabled by default

File: src/aks-preview/HISTORY.rst [hunk 2]
```diff
@@ -13,6 +13,10 @@ Pending
 +++++++
 * Minimise the roles needed to introduce for Elastic SAN for enabling Azure Container Storage with elasticSan storagepool type.
 
+3.0.0b6
++++++++
+* Add option `WindowsAnnual` to `--os-sku` for `az aks nodepool add`.
```
Comment (FumingZhang) [inline src/aks-preview/HISTORY.rst:18]: If you want to release a new version, please move the note from `Pending` (several lines above) here and update `VERSION` correspondingly in `setup.py`.

Comment (junjiezhang1997) [reply src/aks-preview/HISTORY.rst:17]: Updated. Thanks for the comment.

**Review by yanzhudd — APPROVED** [review]

**Pull request merged**